components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/revoke/OAuthRevocationEndpoint.java

```diff
@@ -292,7 +292,7 @@ private Response handleAuthorizationFailure(String callback)
                     .setErrorDescription("Client Authentication failed.").buildJSONMessage();
             return Response.status(response.getResponseStatus())
                     .header(HTTP_RESP_HEADER_AUTHENTICATE, getRealmInfo())
-                    .header(HttpHeaders.CONTENT_TYPE, TEXT_HTML)
+                    .header(HttpHeaders.CONTENT_TYPE,"application/json")
```

Comment:
Use an existing constant.

Reply:
Is it good to define a new constant in the same file or using the AllOWED_CONTENT_TYPE array?